### PR TITLE
Blog post - women in tech

### PR DESCRIPTION
**Pull request by lizpope**
### Blog title
My thoughts on Women in Tech

### Blog description
Highlighting our struggle to recruit women, and whether there is anything we can do about it

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+I’ve taken a look at all the applications we’ve had for our engineering roles in the past 12 months, and found the following:
+Of 109 applicants there were only **three women**
+
+If you consider that of those 109 applicants I’ve hired just six* people then the chances of one of those six being a woman is... (please wait while I do maths)... 0.16%. That’s crazy low!
+
+**If those hire statistics seem strange then I should point out that they are slightly skewed by agencies sending lots of unsuitable candidates who didn’t make the first CV sift, however I don’t have post first sift data to hand - so these numbers will have to do for now! (I’ll put together a blog post on our hiring process another time!)*
+
+But regardless of how you read those statistics, three female applicants in 109 is low. Far too low. But it does answer the question of why I haven’t hired a female engineer this year - they just aren’t applying. I can't give a job to someone who doesn't apply. But that raises a more important question - **Why aren’t women applying?**
```
Comment (dancork) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: Starting 2 sentences with `But` in both instances it's probably not needed.

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+author: liz_pope
+categories: recruitment
+tags: women developers engineers recruitment jobs
+title: My thoughts on Women in Tech
+excerpt: 'This year I’ve done a lot of recruitment and all of the engineers I’ve recruited have been male. Why?'
+---
+
+I was at a conference this week and looking round the room I noticed (once again) a lot of men. I’m going to guess at it being an 85:15 split men to women (though the speaker split seemed more even - credit where credit is due). It got me thinking again about the gender issues in our industry and gave me the push I’ve needed to finally write a blog about it.
+
+In our company we have over 30 full time developers/engineers across our two web teams. Less than five are women. Even our Testing team (a role which I've always found to be more evenly balanced) is nearly three quarters male. I'm responsible (to a large extent) for the make up of the team, this year I’ve done a lot of recruitment and all of the engineers I’ve recruited have been male. Why? Am I sexist? (Against my own sex!) No, of course not! **So why no women?**
```
Comment (dancork) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: Fewer than (not less than)

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+That question is an entire blog post in its own right but my gut feeling says it’s because of a lack of female engineers in general. Girls 10 years ago didn’t code at school so we don’t see them in tech today (I myself dropped my Computer Science class because the teacher didn’t give me the support to catch up to my male classmates). We need to educate girls from a young age, and to support those past school age with an interest in re-training through grants and accessible training programmes (which is beginning to happen).
+
+However, I don't want to assume that there is nothing I can do to encourage more women in to tech and just sit waiting 10 years until the school girls of today are entering work! So I'd love to hear ideas as to how I can encourage more women to apply to Holiday Extras and why they don’t already. Maybe our job adverts don’t seem inclusive, or as a team we don’t come across as female friendly (I hope that’s not true - but I’d rather know so it could be fixed!). So please get in touch if you have an opinion on this. Or if you’re a woman looking for a job then consider applying to us!
```
Comment (dancork) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: into

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md (hunk 2 above)
Comment (lizpope) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: @dancork Fewer/Less always gets me!

**Comment by alixgeorge** [comment]
Taking second review.

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+In our company we have over 30 full time developers/engineers across our two web teams. Fewer than five are women. Even our Testing team, a role which I've always found to be more evenly balanced, is nearly three quarters male. I'm responsible, to a large extent, for the make up of the team and this year I’ve done a lot of recruitment. All of the engineers I’ve recruited have been male. Why? Am I sexist? (Against my own sex!) No, of course not! **So why no women?**
+
+I’ve taken a look at all the applications we’ve had for our engineering roles in the past 12 months, and found the following:
+Of 109 applicants there were only **three women**
+
+If you consider that of those 109 applicants we’ve hired just six* people then the chances of one of those six being a woman is... (please wait while I do maths)... 0.16%. That’s crazy low!
+
+**If those hire statistics seem strange then I should point out that they are slightly skewed by agencies sending lots of unsuitable candidates who didn’t make the first CV sift, however I don’t have post first sift data to hand - so these numbers will have to do for now! (I’ll put together a blog post on our hiring process another time!)*
```
Comment (alixgeorge) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: Would suggest possibly replacing the comma after 'first CV sift' to a full stop and starting a new sentence. I.e. 'However, I don't have post first sift data....'

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+However, I don't want to assume that there is nothing I can do to encourage more women into tech and just sit waiting 10 years until the school girls of today are entering work! So I'd love to hear ideas as to how I can encourage more women to apply to Holiday Extras and why they don’t already. Maybe our job adverts don’t seem inclusive, or as a team we don’t come across as female friendly, I hope that’s not true - but I’d rather know so it could be fixed!. So please get in touch if you have an opinion on this. Or if you’re a woman looking for a job then consider applying to us!
+
+***I'd love to in six months be able to write a blog post publishing further recruitment figures showing an increase in female applicants and to have at least one new female recruit! Help me achieve it!***
```
Comment (alixgeorge) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: Might flow better if syntax is 'In six months I'd love to be able to write a blog post...'

**Comment by lizpope** [comment]
Thanks for the feedback @alixgeorge - I've made a couple of changes based upon it.

File: _posts/2015-11-17-my-thoughts-on-women-in-tech.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+In our company we have over 30 full time developers/engineers across our two web teams. Fewer than five are women. Even our Testing team, a role which I've always found to be more evenly balanced, is nearly three quarters male. I'm responsible, to a large extent, for the make up of the team and this year I’ve done a lot of recruitment. All of the engineers I’ve recruited have been male. Why? Am I sexist? (Against my own sex!) No, of course not! **So why no women?**
+
+I’ve taken a look at all the applications we’ve had for our engineering roles in the past 12 months, and found the following:
+Of 109 applicants there were only **three women**
+
+If you consider that of those 109 applicants we’ve hired just six* people then the chances of one of those six being a woman is... (please wait while I do maths)... 0.16%. That’s crazy low!
+
+**If those hire statistics seem strange then I should point out that they are slightly skewed by agencies sending lots of unsuitable candidates who didn’t make the first CV sift. I don’t have post first sift data to hand - so these numbers will have to do for now! (I’ll put together a blog post on our hiring process another time!)*
```
Comment (dancork) [inline _posts/2015-11-17-my-thoughts-on-women-in-tech.md]: Brackets around the final sentence are unnecessary

**Comment by lizpope** [comment]
I've actually spoken to People Team about getting more accurate stats. So if I get these in the next few days I'll add them to the post before it goes out. If not I'll stick with what I've got.

**Comment by dancork** [comment]
@lizpope sounds cool, once the last comment is addressed I'm happy for it to go :shipit:

**Comment by lizpope** [comment]
@dancork thanks for reviewing, last comment has been addressed. I'll wait for @alixgeorge to take another look and then should be able to set it ready to go live. I'll wait a few days before merging, so it's staggered after the most recent one.

**Comment by alixgeorge** [comment]
Good read @lizpope - +1 from me.

**Pull request merged**